lib/internal/process/task_queues.js

```diff
@@ -151,10 +147,6 @@ function runMicrotask() {
     const callback = this.callback;
     try {
       callback();
-    } catch (error) {
-      // runInAsyncScope() swallows the error so we need to catch
-      // it and handle it here.
-      triggerUncaughtException(error, false /* fromPromise */);
     } finally {
       this.emitDestroy();
```

Comment:
Do we have a check to verify that this `emitDestroy()` is still reached?

Reply:
Yes, `test/parallel/test-queue-microtask-uncaught-asynchooks.js`